src/modules/tests/test-data/cat-a-mod2/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from './test-data.cat-a-mod2.selector';
```

Comment:
Thought we discussed that we are not going to do the barrel files just now. Did this change?

Reply:
I think this was branched off of Ed's barrel file PR

Reply:
Yes; however I am removing it from here so we can merge this work without barrels